external/fv3fit/tests/test_sklearn_wrapper.py

```diff
@@ -52,28 +55,21 @@ def test_flatten_same_order():
     np.testing.assert_allclose(a, b)
 
 
-@pytest.fixture
-def test_regressor_ensemble():
-    base_regressor = LinearRegression()
-    ensemble_regressor = _RegressorEnsemble(base_regressor)
+def test_ensemble_fit():
+    regressor_ensemble = _RandomForestEnsemble(RandomForestHyperparameters())
     num_batches = 3
     X = np.array([[1, 1], [1, 2], [2, 2], [2, 3]])
     y = np.dot(X, np.array([1, 2])) + 3
     for i in range(num_batches):
-        ensemble_regressor.fit(X, y)
-    return ensemble_regressor
-
-
-def test_ensemble_fit(test_regressor_ensemble):
-    regressor_ensemble = test_regressor_ensemble
+        regressor_ensemble.fit(X, y)
     assert regressor_ensemble.n_estimators == 3
     X = np.array([[1, 1], [1, 2], [2, 2], [2, 3]])
     y = np.dot(X, np.array([1, 2])) + 3
     regressor_ensemble.fit(X, y)
     # test that .fit appends a new regressor
     assert regressor_ensemble.n_estimators == 4
     # test that new regressors are actually fit and not empty base regressor
-    assert len(regressor_ensemble.regressors[-1].coef_) > 0
+    assert len(regressor_ensemble.regressors[-1].estimators_) > 0
```

Comment:
The test was refactored to use a random forest instead of a DummyRegressor as part of making the sklearn code random-forest-specific.